Dockerfile

```diff
@@ -35,7 +35,7 @@ COPY --chown=elife:elife app-defaults.cfg ${PROJECT_FOLDER}/
 
 USER root
 RUN mkdir .data && chown www-data:www-data .data
-RUN mkdir logs && chown www-data:www-data logs
+RUN mkdir logs && chown www-data:www-data logs && chmod -R a+w logs
```

Comment:
It will be `www-data` and `elife` writing to the logs

Reply:
Shouldn't it be just `www-data` if we run everything in the container? But fine to keep this backward compatibility if needed.

Reply:
The image is (or would be) used to run the server as well as the "ETL" scripts (migrate schema, load data etc.). The latter is currently meant to be run via the `elife` user. But we could use `www-data` for both if you think that would be better.